services/user.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(username: str,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one